tests/widget/modal/widget-modal.us_es.test.js

```diff
@@ -1,19 +1,16 @@
 import { expect, test } from '@playwright/test';
 
-test('modal widget in US Spanish', async ({ browser }) => {
-  const context = await browser.newContext({ locale: 'es-US' });
-  const page = await context.newPage();
-
+test.use({ locale: 'es-US' });
+test('modal widget in US Spanish', async ({ page }) => {
   await page.goto('widget/modal');
+
   const loginButton = page.locator('button', { hasText: 'Open Login Modal' });
-  const dialog = page.locator('dialog');
-  expect(await dialog.isVisible()).toBeFalsy();
 
   await loginButton.click();
-  expect(await dialog.isVisible()).toBeTruthy();
```

Comment:
These `isVisible` assertions are just a repeat from the base modal test, so removing them for efficiency.